site/kube-addon.md

```diff
@@ -22,6 +22,13 @@ with a single command:
 kubectl apply -f https://git.io/weave-kube
 ```
 
+**Note:** If you are using Kubernetes 1.6 or later, please use this
+  command instead:
+
+```
+kubectl apply -f https://git.io/weave-kube-1.6
+```
```

Comment:
Suggestion:
```
Weave Net can be installed onto your CNI-enabled Kubernetes cluster
with a single command:

* Kubernetes versions up to `1.5.*`:

    kubectl apply -f https://git.io/weave-kube

* Kubernetes versions `1.6` and above:

    kubectl apply -f https://git.io/weave-kube-1.6
```
given it is more a choice for the end user than a note which may seem optional.